ui/app/ot/directives/otCalendarSurgicalAppointment.js

```diff
@@ -1,9 +1,15 @@
 'use strict';
 
 angular.module('bahmni.ot')
-    .directive('otCalendarSurgicalAppointment', ['surgicalAppointmentHelper', function (surgicalAppointmentHelper) {
+    .directive('otCalendarSurgicalAppointment', ['surgicalAppointmentHelper', 'appService', '$window', function (surgicalAppointmentHelper, appService, $window) {
         var link = function ($scope) {
             $scope.attributes = surgicalAppointmentHelper.getSurgicalAttributes($scope.surgicalAppointment);
+            var patientUrls = appService.getAppDescriptor().getConfigValue("patientDashboardUrl");
+            $scope.patientDashboardUrl = patientUrls && patientUrls.link && patientUrls.link.replace("{{patientUuid}}", $scope.surgicalAppointment.patient.uuid);
```

Comment:
for this, we can use existing appService method like this
`appService.getAppDescriptor().formatUrl(patientUrls.link, {'patientUuid': $scope.surgicalAppointment.patient.uuid})`